content/copilot/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Docker for GitHub Copilot
+description: Learn how to use the Docker Extension for GitHub Copilot to streamline Docker-related tasks.
+keywords: Docker, GitHub Copilot, extension, Visual Studio Code, chat, ai, containerization
+---
+
+> **Early Access**
+>
+> The Docker for GitHub Copilot extension is a part of the [Limited Public Beta](https://github.blog/changelog/2024-05-21-copilot-extensions-now-in-limited-public-beta/)
+> for GitHub Copilot Extensions. Only members of the beta program are eligible to use this extension.
+{ .restricted }
+
+The Docker Extension for GitHub Copilot integrates Docker's capabilities with
+GitHub Copilot, providing assistance with containerizing applications and
+generating Docker assets. This extension helps you streamline Docker-related
+tasks wherever GitHub Copilot Chat is available.
+
+The key features are:
+
+- Ask questions and receive responses about containerization in any context
+  when GitHub Copilot Chat is available, such as on GitHub.com and in Visual Studio Code.
+- Automatically generate Dockerfiles, Docker Compose files, and .dockerignore
+  files for a project.
+- Open pull requests with generated Docker assets directly from the chat
+  interface.
+- Get summaries of project vulnerabilities from Docker Scout and receive next
+  steps via CLI.
+
+## Supported languages
+
+Currently, the Docker Extension for GitHub Copilot supports the following
+programming languages:
+
+- Dockerfile
+- Java
+- JavaScript
+- Python
+- TypeScript
+
+## Get Started
+
+Here's how to get started with the Docker Extension for GitHub Copilot.
+
+### Prerequisites
+
+Before you start, ensure that:
+
+- You have been granted access to GitHub's [limited public beta program](https://github.blog/changelog/2024-05-21-copilot-extensions-now-in-limited-public-beta/).
+- The [Docker for GitHub Copilot](https://github.com/marketplace/docker-for-github-copilot)
+  app is installed in your GitHub organization.
+
+### Setup
+
+Select where you want to use the Docker Extension for GitHub Copilot:
+
+{{< tabs >}}
+{{< tab name="Editor or IDE" >}}
+
+For instructions on how to use the Docker Extension for GitHub Copilot in
+your editor, see:
+
+- [Visual Studio Code](https://docs.github.com/en/copilot/github-copilot-chat/copilot-chat-in-ides/using-github-copilot-chat-in-your-ide?tool=vscode)
+- [Visual Studio](https://docs.github.com/en/copilot/github-copilot-chat/copilot-chat-in-ides/using-github-copilot-chat-in-your-ide?tool=visualstudio)
+- [JetBrains IDEs](https://docs.github.com/en/copilot/github-copilot-chat/copilot-chat-in-ides/using-github-copilot-chat-in-your-ide?tool=jetbrains)
```

Comment:
Let's remove the JetBrains note for now.

It looks like it's limited to VS, VS Code and github.com right now,  based on their docs
https://docs.github.com/en/copilot/github-copilot-chat/github-copilot-extensions/about-github-copilot-extensions